src/core_editor/editor.rs

```diff
@@ -17,13 +17,17 @@ impl Default for Editor {
             cut_buffer: Box::new(get_default_clipboard()),
 
             // Note: Using list-zipper we can reduce these to one field
-            edits: vec![],
+            edits: vec![LineBuffer::new()],
             index_undo: 2,
         }
     }
 }
 
 impl Editor {
+    pub fn get_edits(self) -> Vec<LineBuffer> {
```

Comment:
Is this method only used for testing? or are there any other use cases for this?

If it is the former then I would recommend not adding this method at all. The reason for that being: any addition for the public interface ties us to a contract which can make it harder to change the internal details of the particular sturct/class/object/etc. 

For testing we can simply write a unit test in this module which will not require us to add any method to the public interface. 

Thoughts?

Reply:
Propose your change in a branch and if it's a better solutions i will modify this PR, thanks.